mmv1/products/backupdr/BackupPlanAssociation.yaml

```diff
@@ -0,0 +1,137 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BackupPlanAssociation'
+# Remove this towards GA. b/371938375
+min_version: beta
+base_url: projects/{{project}}/locations/{{location}}/backupPlanAssociations
+create_url: projects/{{project}}/locations/{{location}}/backupPlanAssociations/?backup_plan_association_id={{backup_plan_association_id}}
+self_link: projects/{{project}}/locations/{{location}}/backupPlanAssociations/{{backup_plan_association_id}}
+immutable: true
+delete_url: projects/{{project}}/locations/{{location}}/backupPlanAssociations/{{backup_plan_association_id}}
+description: A Backup and DR BackupPlanAssociation.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/backup-disaster-recovery/docs'
+  api: 'https://cloud.google.com/backup-disaster-recovery/docs/reference/rest'
+autogen_async: true
+timeouts:
+  insert_minutes: 20
+  delete_minutes: 20
+examples:
+  - min_version: beta
+    name: 'backup_dr_bpa'
+    primary_resource_id: 'my-backup-plan'
+    vars:
+      project: 'my-project'
+      location: 'us-central1'
+      backup_vault_name: 'bv1'
+      backup_plan_name: 'my-backup-plan'
+      backup_rule_name: 'rule-1'
+      backup_retention_days: 5
+      recurrence_type: 'HOURLY'
+      hourly_frequency: 6
+      time_zone: "UTC"
+      start_hour_of_day: 5
+      end_hour_of_day: 20
+      backup_plan_association_name: 'bpa1'
+      resource_name: 'projects/gcp-project/locations/us-central1/{instance_id}'
```

Comment:
This list seems to show the output_only fields as well. You might want to revisit this.

Reply:
fixed by adding field values in example file onkly